lib/main.js

```diff
@@ -178,13 +197,22 @@ module.exports = class Router extends EventEmitter {
                     sink: this.sink,
                     feedIds,
                     uri: this.buildUri('bundle', req.headers.host, req.secure),
-                    type: req.params.type,
+                    type,
                 });
                 this.emit(
                     'info',
                     `Requested asset bundle produced and successfully uploaded.`
                 );
 
+                if (id) {
+                    const filename = `${type}/${id}`;
```

Comment:
in the case of bundling, the id will be the id of the layout server and in the case of feed upload, id will be the id of the podlet server... right?

Reply:
That's the plan. I wonder if layout id is both the page/layout and the id. Not sure